From 44e841051d51d3b39acb5ce584e84e65358b61dd Mon Sep 17 00:00:00 2001
From: tgeisler <tgeisler@2eb42a87-3827-0410-b767-dc15055982e4>
Date: Mon, 12 Sep 2011 13:55:08 +0000
Subject: [PATCH 051/462] GiT: SetEnv.bat - use regex from boost
 1.31.0-STL-Threadsafe instead of old standalone regex v0.9.9.9 The following
 environment vars are changed: DIR_REGEX INC_REGEX LIB_REGEX

git-svn-id: svn://localhost/SelfServiceBuild/trunk@74 2eb42a87-3827-0410-b767-dc15055982e4
---
 SelfServiceBuild/Build/scripts/setenv.bat | 11 +++++++----
 1 file changed, 7 insertions(+), 4 deletions(-)

diff --git a/SelfServiceBuild/Build/scripts/setenv.bat b/SelfServiceBuild/Build/scripts/setenv.bat
index d96cf15..250b404 100644
--- a/SelfServiceBuild/Build/scripts/setenv.bat
+++ b/SelfServiceBuild/Build/scripts/setenv.bat
@@ -68,6 +68,13 @@ set DIR_BOOST=%DIR_GLOBALSLIB%\boost\1.31.0-STL-Threadsafe\win32
 set INC_BOOST=-I%DIR_BOOST%\include
 set LIB_BOOST=%DIR_BOOST%\lib\libboost_regex-vc6-mt-1_31.lib %DIR_BOOST%\lib\libboost_filesystem-vc6-mt-1_31.lib
 
+rem set DIR_REGEX=%DIR_GLOBALSLIB%\regex\0.9.9.9\win32
+rem set INC_REGEX=-I%DIR_REGEX%\include
+rem set LIB_REGEX=%DIR_REGEX%\lib\vc6-re300l.lib
+set DIR_REGEX=%DIR_BOOST%
+set INC_REGEX=-I%DIR_REGEX%\include
+set LIB_REGEX=%DIR_REGEX%\lib\libboost_regex-vc6-mt-1_31.lib
+
 set DIR_DIRECTOR=%DIR_GLOBALSLIB%\d7xdk\0.9.9.9
 set INC_DIRECTOR=-I%DIR_DIRECTOR%\include
 
@@ -117,10 +124,6 @@ set DIR_QTSDK=%DIR_GLOBALSLIB%\qtsdk\6.0\win32
 set INC_QTSDK=-I%DIR_QTSDK%\cincludes
 set LIB_QTSDK=%DIR_QTSDK%\libraries\qtmlclient.lib
 
-set DIR_REGEX=%DIR_GLOBALSLIB%\regex\0.9.9.9\win32
-set INC_REGEX=-I%DIR_REGEX%\include
-set LIB_REGEX=%DIR_REGEX%\lib\vc6-re300l.lib
-
 set DIR_SED=%DIR_GLOBALSLIB%\sed\0.9.9.9\win32
 set SED=%DIR_SED%\bin\sed.exe
 
-- 
2.41.0.windows.1

